app/Decsys/Data/Entities/Mongo/Webhook.cs

```diff
@@ -4,11 +4,11 @@ namespace Decsys.Data.Entities.Mongo;
 
 public class Webhook
 {
-    public int Id { get; set; }
+    public string Id { get; set; } = string.Empty;
```

Comment:
the Mongo Entity should be `ObjectId` but all the models (and therefore return values of repository and service methods) should be `string`...

it's about separation between the layers.

The service layer and everything on both sides of it should be using `string` since it can't use mongo or litedb directly as it never knows which one might be in use.

The data layer should use `string` when communicating with the the layers above (i.e. in models, or arguments, or return values) but should use the data provider (in this case mongo) native type when interacting with the database.